openml/runs/functions.py

```diff
@@ -85,6 +81,19 @@ def _run_task_get_arffcontent(model, task, class_labels):
             testY = Y[test_indices]
 
             model.fit(trainX, trainY)
+            if isinstance(model, BaseSearchCV):
+                for itt_no in range(0, len(model.cv_results_['mean_test_score'])):
+                    # we use the string values for True and False, as it is defined in this way by the OpenML server
+                    selected = 'false'
+                    if itt_no == model.best_index_:
+                       selected = 'true'
+                    test_score = model.cv_results_['mean_test_score'][itt_no]
+                    arff_line = [rep_no, fold_no, itt_no, test_score, selected]
+                    for key in model.cv_results_:
+                        if key.startswith("param_"):
+                            arff_line.append(str(model.cv_results_[key][itt_no]))
```

Comment:
Hm, we loose lot's of information on which flow a parameter belongs to, is this intended by the way the arff files work here?